### ISSUE TITLE
Validation loss jumps when stopped and reload training

### ISSUE THREAD

**Issue by Steve-Shao** [Steve-Shao/TorchBSDE-Package#1]

**Comment by Steve-Shao** [comment]
It was the random number generator.
